tests/ord-service/tests/api_test.go

```diff
@@ -144,7 +142,8 @@ func TestORDService(stdT *testing.T) {
 		},
 	}
 
-	intSystemHttpClient := integrationSystemClient(t, ctx, unsecuredHttpClient, intSystemCredentials)
+	intSystemHttpClient, err := clients.NewIntegrationSystemClient(ctx, intSystemCredentials)
```

Comment:
For some reason I'm not able to add a comment there, but L138-143 can be deleted